MatrixSDK/Crypto/CryptoMachine/MXCryptoRequests.swift

```diff
@@ -106,15 +106,22 @@ struct MXCryptoRequests {
         }
     }
     
+    @MainActor
+    // Calling methods on `MXRoom` has various state side effects so should be called on the main thread
     func roomMessage(request: RoomMessageRequest) async throws -> String? {
-        var event: MXEvent?
-        return try await performCallbackRequest {
+        return try await withCheckedThrowingContinuation { cont in
```

Comment:
```suggestion
        try await withCheckedThrowingContinuation { continuation in
```
**Optional**
- remove return when it's not needed
- avoid abbreviations like `cont`